build.assets/Makefile

```diff
@@ -88,6 +88,35 @@ PULL_ON_CI = $(if $(filter $(CI),true),docker inspect --type=image $(1) >/dev/nu
 # more details.
 RELEASE_TARGET ?= release
 
+# This variable extracts the name of the Docker buildx builder that uses the Docker driver.
+#
+# The awk command looks for the first line that has "docker" in the second or third column.
+# We check both columns to account for the possibility that the builder is "selected", in which
+# case the second column will be a * and the third column will be "docker".
+#
+# Second column match:
+#
+# $ docker buildx ls
+# NAME/NODE           DRIVER/ENDPOINT             STATUS   BUILDKIT PLATFORMS
+# container-driver *  docker-container
+#   container-driver0 unix:///var/run/docker.sock inactive
+# default             docker
+#   default           default                     running  20.10.23 linux/arm64, linux/arm/v7, linux/arm/v6
+#
+# Third column match:
+#
+# $ docker buildx ls
+# NAME/NODE           DRIVER/ENDPOINT             STATUS   BUILDKIT PLATFORMS
+# container-driver    docker-container
+#   container-driver0 unix:///var/run/docker.sock inactive
+# default *           docker
+#   default           default                     running  20.10.23 linux/arm64, linux/arm/v7, linux/arm/v6
+DOCKER_BUILDER_NAME=$(shell docker buildx ls 2>/dev/null | awk '$$2 == "docker" || $$3 == "docker" {print $$1}')
```

Comment:
nit: can you put spaces around the `=` - it's a little easier to read and Makefiles are tolerant of spaces there as opposed to shell scripts.

Also, you could hack the field separator in awk like this:

    awk -F '[ *]+' '$$2 == "docker" {print $$1}'

with a comment along the lines of "there may be an asterisk between the first and second column if that record is the default, so just subsume the asterisk into the field separator."

It is a shame that `docker buildx ls` is documented to take a `--format` arg, but does not actually take it (at least on my version of docker) - it would make this cleaner.

Reply:
```
awk -F '[ *]+' '$$2 == "docker" {print $$1}'
```

nice one!

> It is a shame that docker buildx ls is documented to take a --format arg, but does not actually take it (at least on my version of docker) - it would make this cleaner.

Yeah, mine neither. Docker is a mess as an overall project.

Reply:
9c41a6810d1eedbf1a9f170ba4c7e8ed6075c288